src/03_modules.py

```diff
@@ -10,22 +10,23 @@
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
-
+print(sys.argv)
```

Comment:
# Print out the command line arguments in sys.argv, one per line

One per line is a sign that you should probably use a loop to get each item in the list.